code/modules/food/recipes_grill.dm

```diff
@@ -0,0 +1,231 @@
+/datum/recipe/humanburger
+	appliance = GRILL
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/meat/human,
+		/obj/item/weapon/reagent_containers/food/snacks/bun
+	)
+	result = /obj/item/weapon/reagent_containers/food/snacks/human/burger
+
+/datum/recipe/plainburger

```

Comment:
Since there's so many types of burgers, would it be possible to organize them similar to the jellydonut/slimedonut, with appliance inherited? That or the slimedonut's appliance should still be specified explicitly for consistency, at which point the inheritance is largely meaningless (I'm also of the opinion that burgers, pizza, etc should be refactored similarly but that may be out of scope)

Reply:
Alternatively to (or perhaps even in conjunction with) the previous suggestion, you might also subtype all appliance-specific recipes so that their appliances are specified by the path? Though that would make multi-appliance recipes a bit inconsistent with having to specify the appliances directly, and thus needing to inherit from the base type for consistency

Reply:
Potentially, but I'd rather look into that later when I do a full refactor of recipes and/or flavor system.